engine/consensus/matching/engine.go

```diff
@@ -33,7 +33,7 @@ import (
 
 // DefaultRequiredApprovalsForSealConstruction is the default number of approvals required to construct a candidate seal
 // for subsequent inclusion in block.
-const DefaultRequiredApprovalsForSealConstruction = 0
+const DefaultRequiredApprovalsForSealConstruction = 1
```

Comment:
https://github.com/onflow/flow-go/commit/7c58991a685b7e2e9b4989c5c875adcc957dc59a

Would I revert this?